### DISCUSSION TITLE
请问可以把1.0的自动合并图集加回来吗

### DISCUSSION THREAD

**Discussion by shanxiaoda** [layabox/LayaAir#1355] [General]
1.0的时候 会自动把碎图合并成一个图集 以此来降低drawcall
2.0给去掉了
但是这个功能真的超级好用！
试想一个农场，里面有很多的农作物 都是碎图并且排列非常乱，导致drawcall很高，如果自动合图后理想状态下drawcall会成为1
或者某个场景 里面几百个怪 但是怪的面积都很小 这时候合并一下 效果也很好
强烈建议加回来

**Comment by LayaCharley** [comment]
LayaAir2肯定回不来了，LayaAir3我们有空的时候再努力努力，1.x的方案是有缺陷的，看看能不能找到一个完美的方案。
